dist/gcc-compatible/Hacl_P256.c

```diff
@@ -592,11 +592,19 @@ static void montgomery_multiplication_buffer(uint64_t *a, uint64_t *b, uint64_t
   uint64_t t[8U] = { 0U };
   uint64_t round2[8U] = { 0U };
   uint64_t round4[8U] = { 0U };
+  uint32_t size_0 = (uint32_t)0U;
```

Comment:
should we mark `size_i` with `[@inline_let]`?

Reply:
done